config/locales/en.yml

```diff
@@ -21,3 +21,4 @@
 
 en:
   hello: "Hello world"
+  badge: "Crachá"
```

Comment:
@frmendes Changing the default locale back to `pt`, using the old pt.yml file and adding badge translation seemed to work. I think this is the best option since it also fixes #85 that occurred right after changing the locale to `en`.